src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketSCMSourceBuilder.java

```diff
@@ -66,11 +71,12 @@ public class BitbucketSCMSourceBuilder extends SCMSourceBuilder<BitbucketSCMSour
      */
     public BitbucketSCMSourceBuilder(@CheckForNull String id, @NonNull String serverUrl,
                                      @CheckForNull String credentialsId, @NonNull String repoOwner,
-                                     @NonNull String repoName) {
+                                     @NonNull String repoName, @CheckForNull String mirrorId) {
```

Comment:
This is an incompatible change and I think it broke Blue Ocean: https://github.com/jenkinsci/bom/pull/2922#issuecomment-1941333185

Either switch the `mirrorId` to a (possibly “fluid”) setter, or retain a `@Deprecated` constructor with the original signature.